README.md

```diff
@@ -6,7 +6,7 @@ Status: Stage 2
 
 ## Synopsis
 
-Top-level `await` enables modules to act as big async functions: With top-level `await`, modules can `await` resources, causing other modules who `import` them to wait before they start evaluating their body.
+Top-level `await` enables modules to act as big async functions: With top-level `await`, ESM modules can `await` resources, causing other modules who `import` them to wait before they start evaluating their body.
```

Comment:
I don’t understand the clarification. It doesn’t make sense to think that HTML or JSON modules could `await` anything, let alone at the top-level. What other modules are there?

Reply:
surely wasm modules could, conceptually, but since they're not JS, this entire proposal wouldn't really apply to them?

Reply:
For Node.js users "modules" could mean CommonJS modules as well. This sends the message "you can stop reading if you only use CommonJS modules".

Reply:
> surely wasm modules could, conceptually, but since they're not JS, this entire proposal wouldn't really apply to them?

Exactly!

> For Node.js users "modules" could mean CommonJS modules as well. This sends the message "you can stop reading if you only use CommonJS modules".

This is an ECMAScript proposal, and “module” has a well-defined meaning within ECMAScript. The previous sentence was clear and precise. The change only adds confusion, IMHO (“ESM” is not an ECMAScript term).

Reply:
typically i've seen `Module` mean the ESM parse goal, and `module` mean the concept - maybe it'd be better to correct these to say `Module`?

Reply:
I bet capitalizing it wouldn't be enough clarification for people who see the word "module" and think "this is ambiguous between CJS and ESM".

I would be happy to take suggestions for tweaking the wording, but this document will be read by people will many kinds of backgrounds, so I would like to make it as explicit as possible.